.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ jobs:
     - checkout
     - restore_cache:
         keys:
-        - v1.4-{{checksum "go.sum"}}-{{checksum "sdk/go.sum"}}-{{checksum "api/go.sum"}}
+        - v1.5-{{checksum "go.sum"}}-{{checksum "sdk/go.sum"}}-{{checksum "api/go.sum"}}
```

Comment:
Why does the cache version need to change?

Reply:
If any caches were uploaded in the last few days without this `GOPATH` fix, they would be missing modules, so might cause future builds to break.

Reply:
Oh right, the cache are still being stored, they're just empty.